MAINTAINERS.md

```diff
@@ -83,3 +83,37 @@ important maintenance task. When performing the update, follow this procedure:
    prevent a CI failure in case any flags changed in the PR or upstream.
 1. Create a PR with a title like `Update Envoy to 9753819 (Jan 24th 2021)`,
    describe all performed changes in the PR's description.
+
+## Identifying an Envoy commit that introduced a breakage
+
+### Background
+
+Sometimes CI tests fail after updating Nighthawk to an Envoy dependency. If the
+root cause is hard to be identified, the bisect could be worth a try.
+
+### How to bisect
+
+The bisect is used to find the problematic Envoy commit between the commit from
+the last successful Nighthawk update and the current commit that Nighthawk needs
+to be updated to.
+
+Following the
+[update process](https://github.com/envoyproxy/nighthawk/blob/main/MAINTAINERS.md#updates-to-the-envoy-dependency)
```

Comment:
(nit) If we can, we should prefer relative links, as clicking on those stays on the same branch.

Reply:
Done.